src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/BundleController.kt

```diff
@@ -0,0 +1,60 @@
+package org.vaccineimpact.orderlyweb.controllers.api
+
+import com.google.gson.Gson
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.controllers.Controller
+import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.db.Config
+import org.vaccineimpact.orderlyweb.errors.MissingParameterError
+import org.vaccineimpact.orderlyweb.errors.OrderlyServerError
+
+class BundleController(
+        context: ActionContext,
+        config: Config,
+        private val httpClient: OkHttpClient
+) : Controller(context, config)
+{
+    @Suppress("unused")
+    constructor(context: ActionContext) :
+            this(
+                    context,
+                    AppConfig(),
+                    OkHttpClient()
+            )
+
+    fun pack(): Boolean
+    {
+        val name = context.params(":name") ?: throw MissingParameterError("name")
+
+        val url = appConfig["orderly.server"] + "/v1/bundle/pack/$name" + (if (context.queryString() != null) "?" + context.queryString() else "")
```

Comment:
As mentioned, can we move all this logic to the `OrderlyServer` class?

A pedantic style point, our convention is to wrap all conditionals in braces, but I would make this more kotlin-esque as follows:

`val url = "${appConfig["orderly.server"]}/v1/bundle/pack/$name?${context.queryString()?: "" }"`

(^ does add an extra "?" even when no query string, but that seems harmless enough)